src/Rules/ZipCode.php

```diff
@@ -6,6 +6,13 @@
 
 class ZipCode implements Rule
 {
+    public $parameters;
```

Comment:
Set the default value.

```suggestion
    public $parameters = [
        'strict' => false
    ];
```